backend/de.metas.async/src/main/java/de/metas/async/processor/impl/WorkpackageProcessorTask.java

```diff
@@ -600,18 +599,11 @@ private void markError(final I_C_Queue_WorkPackage workPackage, final AdempiereE
 
 		//
 		// Allow retry processing this workpackage?
-		if (workPackageProcessorWrapped.isAllowRetryOnError())
-		{
-			workPackage.setProcessed(false); // just in case it was true
-		}
-		else
-		{
-			// Flag the workpackage as processed in order to:
-			// * not allow future retries
-			// * avoid discarding items from this workpackage on future workpackages because they were enqueued here
-			// TODO shall we also release the elements lock if any?
-			workPackage.setProcessed(true);
-		}
+		// Flag the workpackage as processed in order to:
+		// * not allow future retries
+		// * avoid discarding items from this workpackage on future workpackages because they were enqueued here
+		// TODO shall we also release the elements lock if any?
+		workPackage.setProcessed(!workPackageProcessorWrapped.isAllowRetryOnError()); // just in case it was true
```

Comment:
the `// just in case it was true` does not make sense anymore
also, the whole comments needs to be re-aranged...
e.g. `// Allow retry processing this workpackage?` makes no sense there neither...